counterfit/frameworks/textattack/textattack.py

```diff
@@ -1,69 +1,49 @@
-import re
-import importlib
+import yaml
+import glob
+import pathlib
+import numpy as np
 
 from textattack import Attacker
-import numpy as np
 from textattack.datasets import Dataset
 
 from counterfit.core.attacks import CFAttack
-from counterfit.core.frameworks import Framework
-from counterfit.core.utils import get_subclasses
-from counterfit.report.report_generator import get_target_data_type_obj
+from counterfit.core.frameworks import CFFramework
 
-class TextAttackFramework(Framework):
+
+class TextAttackFramework(CFFramework):
```

Comment:
`DeepWordBug` attack failed on target `movie_reviews`
![image](https://user-images.githubusercontent.com/43563047/199325887-9df77129-e1dd-4b1e-8af3-141076425e55.png)

Reply:
Fixing this was way more complicated than it seems... fixed 02c8acce2737bf0bae543b57bbe5d65fa9a19b50. 👍

Reply:
Tested and issue resolved.